vetapp/app/controllers/visits_controller.rb

```diff
@@ -0,0 +1,53 @@
+class VisitsController < ApplicationController
+       before_action :authenticate_user!
+
+    def index
+        @animal=Animal.all
+        @visit=Visit.all
+        @user=User.all
+    end
+    
+    def show
+    end
+    
+    def new
+        @visit=Visit.new
+        @animal=Animal.all
+        @user=User.all
+    end
```

Comment:
Do you need these instance variables here? Are you using them in the new visit view?